source/core.py

```diff
@@ -362,10 +361,10 @@ def set_performance():
     print("\nTotal CPU usage:", cpuload, "%")
     print("Total system load:", load1m, "\n")
 
-    if load1m >= CPUS / 5:
+    if load1m >= (10*CPUS)/100:
```

Comment:
Could someone explain me, what does `load1m` mean?

Reply:
"_It conventionally appears in the form of three numbers which represent the system load during the last one-, five-, and fifteen-minute periods._" - Reference: https://en.wikipedia.org/wiki/Load_(computing)

`load1m` var is ought to represent last minute.

Reply:
oh, I see its weighted average of system load with window size of 1 minute. Thanks